django/applications/catmaid/control/cropping.py

```diff
@@ -481,7 +464,8 @@ def extract_substack_no_rotation(job) -> List:
 
             # Write out the image parts and make sure the maximum allowed file
             # size isn't exceeded.
-            cropped_slice = Image(Geometry(bb.width, bb.height), ColorRGB(0, 0, 0))
+
+            cropped_slice = PILImage.new(bb.width, bb.height)
```

Comment:
This doesn't work and I believe it needs to look like this:
```
cropped_slice = PILImage.new(mode="RGB", size=(bb.width, bb.height))
```
This also enforced that all image patches that we load are internally treated as RGB (for now), which is an assumption in other places of the cropping code.

Reply:
Since I merged this branch already, I fixed this in `dev`.